sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md

```diff
@@ -2,6 +2,7 @@
 
 ## Current version
 ###  Feature Additions
+- Add Switch activity
```

Comment:
Also add "- Added new AzureMLExecutePipeline activity and AzureMLService linked service type